tools/autograd/derivatives.yaml

```diff
@@ -560,12 +589,13 @@
 
 - name: index.Tensor(Tensor self, Tensor?[] indices) -> Tensor
   self: index_backward(grad.new_zeros(self.sizes(), self.options()), indices, grad)
-  indices: TensorList()
```

Comment:
?

Reply:
These inputs cannot require gradients. So there is no reason to put anything here.
I can move that change to the derivatives.yaml cleanup PR if you want though?